test/jdk/javax/swing/JTextPane/DefaultStyledDocumentTest.java

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Guess it can be moved to test/jdk/javax/swing/text/DefaultStyledDocument/
Also, rename it something like TestDocNegLenCharAttr or something similar.

Reply:
Updated.